CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.8.2
+
+### Calc Functions Interpolation Migrator
+
+* Add parenthesis in operations to keep order bug fix.
```

Comment:
A clearer wording here would probably be something like "Add parentheses in place of interpolation when necessary to preserve the evaluation order"